packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -672,75 +672,157 @@ void main() {
 
     await runner.run(<String>['create', '--no-pub', projectDir.path]);
 
-    expect(projectDir.childDirectory('linux').childFile('CMakeLists.txt').existsSync(), true);
+    expect(projectDir.childDirectory('linux').existsSync(), false);
+    expect(projectDir.childDirectory('macos').existsSync(), false);
+    expect(projectDir.childDirectory('windows').existsSync(), false);
+    expect(projectDir.childDirectory('web').existsSync(), false);
   }, overrides: <Type, Generator>{
-    FeatureFlags: () => TestFeatureFlags(isLinuxEnabled: true),
+    FeatureFlags: () => TestFeatureFlags(),
   });
 
-  testUsingContext('app does not include Linux by default', () async {
+  testUsingContext('plugin does not include desktop or web by default',
+      () async {
     Cache.flutterRoot = '../..';
     when(mockFlutterVersion.frameworkRevision).thenReturn(frameworkRevision);
     when(mockFlutterVersion.channel).thenReturn(frameworkChannel);
 
     final CreateCommand command = CreateCommand();
     final CommandRunner<void> runner = createTestCommandRunner(command);
 
-    await runner.run(<String>['create', '--no-pub', projectDir.path]);
+    await runner.run(
+        <String>['create', '--no-pub', '--template=plugin', projectDir.path]);
 
-    expect(projectDir.childDirectory('linux').childFile('CMakeLists.txt').existsSync(), false);
+    expect(projectDir.childDirectory('linux').existsSync(), false);
```

Comment:
FYI: this isn't used consistently but you can use `exists` or `isNot(exists)` from `package:file_testing`